tools/psoxy-test/lib/utils.js

```diff
@@ -374,10 +376,19 @@ async function getAWSCredentials(role, region) {
     }
     credentialsProvider = fromTemporaryCredentials(temporaryCredentialsOptions);
 
-    credentials = await credentialsProvider();
+    try {
+      credentials = await credentialsProvider();
+    } catch (e) {
+      throw new Error(`Unable to get AWS credentials: ${e.message}\nMake sure your AWS CLI is configured correctly: https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-authentication.html`);
+    }
 
-    const callerIdentity = JSON.parse(
-      executeCommand('aws sts get-caller-identity').trim());
+    try {
+      callerIdentity = JSON.parse(
+        executeCommand('aws sts get-caller-identity').trim());
+    } catch (e) {
+      // It shouldn't happen if credentials are valid
+      logger.verbose(`Unable to get caller identity: ${e.message}`);
```

Comment:
If that shouldn't happen but, what if happens .. ?

```suggestion
      throw new Error .. ?
```